app/test_main.py

```diff
@@ -1 +1,59 @@
-# write your code here
+import unittest
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+class TestCanAccessGooglePage(unittest.TestCase):
+
+    @mock.patch("app.main.valid_google_url")
```

Comment:
DRY 

use [@pytest.mark.parametrize](https://docs.pytest.org/en/stable/how-to/parametrize.html#pytest-mark-parametrize)